### PR TITLE
darkdump #3010

### PR DESCRIPTION
**Pull request by dualfade**
please review --

### CODE REVIEW

File: packages/darkdump/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+pkgver=15.f3bc7c0
+pkgrel=1
+pkgdesc='Search The Deep Web Straight From Your Terminal.'
+arch=('any')
+groups=('blackarch' 'blackarch-webapp' 'blackarch-scanner')
+url='https://github.com/josh0xA/darkdump'
+license=('MIT')
+depends=('python' 'python-requests' 'python-termcolor' 'python-argparse')
+makedepends=('git' 'python-setuptools')
+source=($pkgname::"git+https://github.com/josh0xA/darkdump.git")
```
Comment (Edu4rdSHL) [inline packages/darkdump/PKGBUILD:14]: If the name of the git repo is already `$pkgname` as in this case you don't need to add `$pkgname::`, use  `source=("git+https://github.com/josh0xA/$pkgname.git")` instead.

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]
Hello, @dualfade! Thank you for the PR, please update according to my comments but I also have another recommendation, please [link pull requests to issues using keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) correctly, that way we don't need to close issues individually as I did yesterday, they got closed once we merge the PR. You can use: Fix #issue_number, for this case:

Fix #3010

File: packages/darkdump/PKGBUILD (hunk 1 above)
Comment (dualfade) [reply packages/darkdump/PKGBUILD:14]: Ah yes no problem. Will do that moving forward. Updating now.

**Comment by Edu4rdSHL** [comment]
Thank you, @dualfade !

**Pull request merged**